sdk/documentintelligence/ai-document-intelligence-rest/review/ai-document-intelligence.api.md

```diff
@@ -2096,7 +2271,6 @@ export interface Routes {
 
 // @public
 export interface SimplePollerLike<TState extends OperationState<TResult>, TResult> {
-    getOperationId(): string;
     getOperationState(): TState;
     getResult(): TResult | undefined;
     isDone(): boolean;
```

Comment:
Why are we GA's deprecated stuff here such as `isDone`, `isStopped`, `stopPolling`, `toString`?

Reply:
this is generated code, I did not touch, should I log an issue at autorest?

Reply:
Yes, I think we should generate these only to maintain backward compatibility for existing stable libraries. Can this code be customized using @timovv's tool?

Reply:
removed `SimplePollerLike`